ci/build_container/build_recipes/fmtlib.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+set -e
+
+VERSION=4.0.0
+
+wget -O fmt-$VERSION.tar.gz https://github.com/fmtlib/fmt/archive/$VERSION.tar.gz
+tar xf fmt-$VERSION.tar.gz
+rsync -av fmt-$VERSION/* $THIRDPARTY_SRC/fmtlib
```

Comment:
Can you escape all the variables here? With `""`, e.g. `"$THIRD_PARTY_SRC"/fmtlib`. This is required by the Google style code (https://google.github.io/styleguide/shell.xml#Quoting) and provides safety when there is WSP in the path.

Reply:
I think this is likely broken in other build_recipes also. Can you fix those too?

Reply:
sure, I can do that